### ISSUE TITLE
Milano, Italy

### ISSUE THREAD

**Issue by sabas** [openaddresses/openaddresses#2660]
As tracked in #2434 I scraped the data and I added the street names from OpenStreetMap.
Some time ago a municipal agency added their reference numbers to the street segments and I used it to join the data.
If you want to grab the zipped shapefile, then we can do the proper json.
http://sabas.land/milano_civici.zip
NUMERO: housenumber
CAP: postcode
CODICE_VIA: their reference
NOME_VIA: street name
ZONA: local admin (in osm is admin_level=10)

**Comment by nvkelso** [comment]
Thanks for this followup. Looks like our current source has address points on the road (versus offset onto buildings). Is this one better?

**Comment by sabas** [comment]
Yes it has it on the accesses (minus the errors), checked versus OSM

**Comment by iandees** [comment]
I feel like I might have asked this already, but is there any reason not to go straight to the Esri server listed in that ticket?

http://217.31.113.169/arcgis/rest/services/DynamicLayer/geoschool_2k14/MapServer/0

**Comment by sabas** [comment]
I took the data from there, but there were no street name information...

**Comment by sergiyprotsiv** [comment]
This is indeed a better source than the national data we have (which is essentially interpolated). Adding this would complete the Lombardia province as we have all other sources from their official page. The only question is whether getting the street names from OSM is fine license-wise.

**Comment by justinelliotmeyers** [comment]
Lombardia should have a open road dataset (Strade) (maybe in here: https://dati.lombardia.it/ )? or Italia should have cadastral/ addresses in the INSPIRE initiative? 
![untitled](https://cloud.githubusercontent.com/assets/5324316/24229335/51ca8076-0f4f-11e7-9414-4242e2280f07.jpg)

**Comment by sabas** [comment]
Nationwide there should be a road cadastre but it's a matter of sci-fi, no one actually saw it ever :-)

Roads from Milano province (now Città Metropolitana) are here http://www.geoportale.regione.lombardia.it/download-ricerca
Database Topografico Regionale - Viabilità, mobilità e trasporti

The problem would become the match with the address points (there's no common key)

**Comment by sabas** [comment]
Milan finally released the addresses as open data
https://geoportale.comune.milano.it/Download/area_download/SIT/Toponomastica/OpenData_Civici.zip

The archive contains some csv files which need to be joined.
